charts/argo-cd/values.yaml

```diff
@@ -12,6 +12,11 @@ global:
     repository: argoproj/argocd
     tag: v1.3.4
     imagePullPolicy: IfNotPresent
+  securityContext:
```

Comment:
This shouldn't need an enabled, just default `securityContext` to `{}`, removed `enabled` and comment out the below `runAsUser`, etc.